target/linux/ramips/base-files/etc/init.d/enablemodem

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh /etc/rc.common
+. /lib/ramips.sh
+
+START=99
+
+start() {
+	local board=$(ramips_board_name)
+	if [ $board = "mr200" ]; then
+		adb shell chmod +x /WEBSERVER/www/cgi-bin/*
+		adb shell httpd -h /WEBSERVER/www/ &
+		echo "2357 000d" > /sys/bus/usb-serial/drivers/option1/new_id
```

Comment:
the last line (the echo line), will make further connection the the LTE device through adb impossible. adb exits with this error: "error: device not found"